src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -384,8 +385,9 @@ async function check_new_access_key_exists(config_fs, action, data) {
  * @param {import('../sdk/config_fs').ConfigFS} config_fs
  * @param {object} data
  * @param {string} action
+ * @param {object} user_input
  */
-async function validate_bucket_args(config_fs, data, action) {
+async function validate_bucket_args(config_fs, data, action, user_input) {
```

Comment:
We want to avoid adding the argument in the function signature (it is not clear when to access `data` and when `user_input`).

Reply:
Bucket name? It is not used. The new notifications field is used (if it is provided).

There's a distinction between data and user_input - data is user_input merged with file's content.
Someone who changes this is supposed to understand this.
I will add a comment to clarify.